### DISCUSSION TITLE
Can we get message highlight color priority over badge highlight color?

### DISCUSSION THREAD

**Discussion by 7JEO** [Chatterino/chatterino2#3204] [Ideas]
I would like the message highlight color to take priority over badge highlight color. I think that its more important to easily see a mention rather than the role the user has. 

**Screenshots**
colors: 
badge: 
![image](https://user-images.githubusercontent.com/79304571/131051789-0225423e-3fc9-423c-9578-5a4f52787097.png)
message:
![image](https://user-images.githubusercontent.com/79304571/131051817-04625030-7c54-42f9-9d1f-b2ea8cdaade8.png)
what it is now: 
![image](https://user-images.githubusercontent.com/79304571/131051761-22786879-b530-4484-970c-390df020e545.png)
what it should be:
![image](https://user-images.githubusercontent.com/79304571/131051901-fad22b0f-8232-41b3-968e-18ff8b91a0c1.png)

**Comment by 7JEO** [comment]
Also i have a broadcaster badge highlight color set and it doesn't highlight me when I type in my own chat. Is that intentional?

**Reply by apa420** [reply]
Chatterino won't highlight your own messages so that's intended @jeovanny315 .
